sdk/communication/communication-identity/review/communication-identity.api.md

```diff
@@ -25,7 +25,7 @@ export class CommunicationIdentityClient {
     createUserAndToken(scopes: TokenScope[], options?: OperationOptions): Promise<CommunicationUserToken>;
     deleteUser(user: CommunicationUserIdentifier, options?: OperationOptions): Promise<void>;
     getToken(user: CommunicationUserIdentifier, scopes: TokenScope[], options?: OperationOptions): Promise<CommunicationAccessToken>;
-    getTokenForTeamsUser(teamsUserAadToken: string, options?: OperationOptions): Promise<CommunicationAccessToken>;
+    getTokenForTeamsUser(options: GetTokenForTeamsUserOptions): Promise<CommunicationAccessToken>;
```

Comment:
Just to double-check, `getTokenForTeamsUser` was not part of the previous GA release right?

Reply:
yes